index.js

```diff
@@ -39,4 +39,75 @@ searchBtnCmp.addEventListener("click", function() {
             weatherWindCmp.appendChild(newWindVal);
         })
         .catch(error => alert("Wrong City Name"))
-});
\ No newline at end of file
+});
+const mondayCmp = document.getElementById("selectMonday");
+const daysList = document.getElementById("getDaysList");
+const days = ['Sun', 'Mon', 'Tue', 'Wed', 'Thur', 'Fri', 'Sat'];
+const nextDays = 4;
+
+function createDays() {
+    let today = new Date();
+    let daysSorted = [];
+    for (let i = 0; i < nextDays; i++) {
+        let newDate = new Date(today.setDate(today.getDate() + 1));
+        daysSorted.push(days[newDate.getDay()]);
+    }
+    for (let i = 0; i < daysSorted.length; i++) {
+        let newLiElement = document.createElement("li");
+        newLiElement.textContent = daysSorted[i];
+        daysList.appendChild(newLiElement);
+    }
+    daysList.style.color = "white";
+    daysList.style.listStyle = "none";
+    daysList.style.padding = "15px";
+};
+createDays();
+
+function generateFiveDayForecast() {
+    const apiUrl = "https://api.openweathermap.org/data/2.5/forecast?q=Munich&units=metric&appid=08a09c29086a3f06cd37337b12b1711f";
+    fetch(apiUrl)
+        .then(response => response.json())
+        .then(data => {
+            let firstDayNewTemp = document.createElement("span");
```

Comment:
try to create a helper function which would generate structure for you, since you have 4 blocks of code duplication.
```js
    let firstDayNewTemp = document.createElement("span");
    firstDayNewTemp.innerHTML = data.list[1].main.temp;
    document.querySelectorAll("ul > li")[0].appendChild(firstDayNewTemp);
    firstDayNewTemp.style.marginLeft = "70px";
    let firstDayIcon = data.list[0].weather[0].icon;
    let firstImageUrl = "http://openweathermap.org/img/wn/" + firstDayIcon + "@2x.png";
    let firstNewIcon = document.createElement("span");
    firstNewIcon.innerHTML = "<img src=" + firstImageUrl + ">";
    document.querySelectorAll("ul > li")[0].appendChild(firstNewIcon);
    firstNewIcon.style.marginLeft = "30px";
```